pkg/api/snapshots.go

```diff
@@ -13,6 +13,8 @@ type SnapshotResponse struct {
 	AddedCounts    map[string]int64 `json:"added_counts"`    // Count of each content type
 	RemovedCounts  map[string]int64 `json:"removed_counts"`  // Count of each content type
 	URL            string           `json:"url"`             // URL to the snapshot's content
+	RepositoryName string           `json:"repository_name"`
+	RepositoryUUID string           `json:"repository_uuid"`
```

Comment:
I'd write a comments here for the repository name and UUID to describe them. Something like "name of associated repository" or "name of repository the snapshot belongs to". You'll also want to regen the api spec so those descriptions get included.

Reply:
fixed [here](https://github.com/content-services/content-sources-backend/pull/796/commits/b93e539c29cf17c4e5adc0233d36a4a86ef66cc2) 👍🏼